modules/mobile/app/models/mobile/user.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Mobile
+  class User < ApplicationRecord
+    validates :icn, presence: true, uniqueness: true
+
+    after_initialize :set_defaults
+
+    def increment_vet360_link_attempts
+      self.vet360_link_attempts += 1
+    end
+
+    private
+
+    def set_defaults
```

Comment:
Yes, I've been bit in the butt quite a few times fighting a hidden database default value. My personal preference is to set the defaults in the model so that there are no surprises. Obviously in a case like this where the default is 0, there's little risk to having the database set the default. But if there are ever more defaults I'd rather have them in the model. I historically never insert/update records without ActiveRecord so the integrity of the data is still checked with the active model validations